test/validation/api/pktio/pktio.c

```diff
@@ -3089,13 +3092,15 @@ static int pktio_check_start_stop(void)
 	return ODP_TEST_ACTIVE;
 }
 
+#include <unistd.h>
```

Comment:
Why?

Reply:
Oops, that's a leftover of some test code that I had in there when making this change. I'll remove it in the next version.